src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -1151,38 +1151,10 @@ def body(self):
 
         # now attach to existing
         self.cmd('vm unmanaged-disk attach -g {} --vm-name {} -n {} --vhd {} --caching ReadOnly'.format(
-            self.resource_group, self.vm_name, disk_name, vhd_uri))
-
-        # make it fun, let us migrate the data disk to managed image
-        self.cmd('vm stop -g {} -n {}'.format(self.resource_group, self.vm_name))
-        self.cmd('vm generalize -g {} --n {}'.format(self.resource_group, self.vm_name), checks=NoneCheck())
-        self.cmd('vm show -g {} --n {}'.format(self.resource_group, self.vm_name))
-        self.cmd('vm capture -g {} -n {} --vhd-name-prefix vmtest'.format(self.resource_group, self.vm_name),
```

Comment:
I removed those test steps. The captured image is not really usable, because the test itself could not ssh and run `waagent -deprovision+user` to cleanly generalize the vm, hence any vm/vmss created from the image could not reboot and the test will fail